src/Loader/Loader.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import CircularProgress from '@mui/material/CircularProgress';
+import { SxProps } from '@mui/material';
 
 export interface LoaderProps {
   color?: string;
+  sx?: SxProps;
   className?: string;
 }
 
-export const Loader: React.FC<LoaderProps> = ({ className }) => (
+export const Loader: React.FC<LoaderProps> = ({ className, sx }) => (
   <div className={className}>
-    <CircularProgress color='primary' />
+    <CircularProgress sx={sx} color='primary' />
```

Comment:
We don't use color here but it's passed in props.

Reply:
Good catch i changed it